### ISSUE TITLE
Search Screen - Departure location

### ISSUE THREAD

**Issue by GrosR** [clowdfish/ttg-frontend#75]
Can we allow to save a departure location in a kind of favorites list that will be displayed once a user enters the field? 

I know this will not work over systems or device borders as we have no user settings area..still, it would add some convenience to the currently used computer

**Comment by davidtran** [comment]
Yes, we can, but in order to implement this feature, we need the backend to support authentication feature and save departure location feature first.

**Comment by GrosR** [comment]
Hi Nam, 
I understand your thoughts and the technical background.. Still, my idea was more to store a local cookie in the users browsers, which would neither be a save nor a sophisticated feature. 

Would that be possible? 

Authentication and user profiles which could really safely store such information shall be part of paid features which we will deliver at a later stage.

**Comment by davidtran** [comment]
Well, I didn't think about it in the first place. Yes, we could store it on browsers :)

**Comment by GrosR** [comment]
:- ) This is an easy way to improve convenience without the whole User-Profile and Authentication/Authorization stuff... :-)
